backend/src/services/mempool_monitor.py

```
@@ -1,257 +1,233 @@
+                                    signature = str(value.signature)
+                                    logs = value.logs
+                                    await self._process_mempool_event(signature, logs)
+                                elif isinstance(event, dict) and 'params' in event:
+                                    result = event['params']['result']['value']
+                                    signature = result['signature']
+                                    logs = result['logs']
+                                    await self._process_mempool_event(signature, logs)
+                            except (AttributeError, KeyError, TypeError) as e:
+                                continue
+
+            except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)